clusterloader2/cmd/clusterloader.go

```diff
@@ -283,8 +283,7 @@ func main() {
 	var prometheusFramework *framework.Framework
 	if clusterLoaderConfig.PrometheusConfig.EnableServer {
 		// Pass overrides to prometheus controller
-		clusterLoaderConfig.TestScenario.OverridePaths = testOverridePaths
-		if prometheusController, err = prometheus.NewController(&clusterLoaderConfig); err != nil {
+		if prometheusController, err = prometheus.NewController(&clusterLoaderConfig, testOverridePaths); err != nil {
```

Comment:
Actually the `testOverridePaths` is global and it'd be totally reasonable to have it as a field in the ClusterLoaderConfig. The benefit is that we won't have to modify these signatures to accept the `testOverridePaths` in addition to `clusterLoaderConfig`